### PR TITLE
Make task

### PR DESCRIPTION
**Pull request by alex39292**

### CODE REVIEW

File: Loops/fourthTask.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+for (let i = 2; i <= 10; i+=2) {
+      console.log( i )
+  }
```
Comment (SiarheiRylach) [inline Loops/fourthTask.js]: fix indentation

File: Loops/sixTask.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let value;
+
+do {
+  value = prompt("Введите число, большее 100?", 0);
+} while (value <= 100);
```
Comment (SiarheiRylach) [inline Loops/sixTask.js]: Please check the solution on learnjs for this task. Looks like you missed part of verification.

File: Loops/fourthTask.js (hunk 1 above)
Comment (alex39292) [reply Loops/fourthTask.js]: Fix

File: Loops/sixTask.js (hunk 2 above)
Comment (alex39292) [reply Loops/sixTask.js]: Yes. I fixed it.

**Comment by alex39292** [comment]
Resolved conflict

**Pull request merged**